packages/Search/examples/distributed_tree_driver/distributed_tree_driver.cpp

```diff
@@ -12,21 +12,126 @@
 #include <DTK_DistributedSearchTree.hpp>
 
 #include <Kokkos_DefaultNode.hpp>
-#include <Teuchos_CommHelpers.hpp>
 #include <Teuchos_CommandLineProcessor.hpp>
-#include <Teuchos_DefaultComm.hpp>
-#include <Teuchos_StandardCatchMacros.hpp>
-#include <Teuchos_Time.hpp>
-#include <Teuchos_TimeMonitor.hpp>
 
+#include <mpi.h>
+
+#include <algorithm>
+#include <chrono>
 #include <cmath> // cbrt
+#include <numeric>
 #include <random>
+#include <utility>
+#include <vector>
+
+// Poor man's replacement for Teuchos::TimeMonitor
+class TimeMonitor
+{
+    using container_type = std::vector<std::pair<std::string, double>>;
```

Comment:
Why not use `std::map<std::string, double>`? Using `vector` would mean that you could get duplicated timers.

Reply:
Meant to keep them in order they had been registered.

Reply:
What benefit does it provide?

Reply:
Chronological order :)